src/migration/1724080448425-ContentLanguage.ts

```diff
@@ -0,0 +1,19 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class ContentLanguage1724080448425 implements MigrationInterface {
+  name = 'ContentLanguage1724080448425';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
```

Comment:
Isn't it already stored somewhere when we get it from yggdrasil?

Reply:
yes, `contentMeta`